diff -up cyrus-sasl-2.1.28/lib/Makefile.am.1~ cyrus-sasl-2.1.28/lib/Makefile.am
--- cyrus-sasl-2.1.28/lib/Makefile.am.1~	2022-06-20 02:43:31.369214260 +0200
+++ cyrus-sasl-2.1.28/lib/Makefile.am	2022-06-20 02:49:28.891502014 +0200
@@ -80,7 +80,8 @@ libobj_la_SOURCES =
 libobj_la_LIBADD = $(LTLIBOBJS)
 
 libsasl2_la_SOURCES = $(common_sources) $(common_headers)
-libsasl2_la_LDFLAGS = -version-info $(sasl_version) -no-undefined
+libsasl2_la_LDFLAGS = -version-info $(sasl_version) -no-undefined -Wl,--version-script=$(top_srcdir)/Versions
+libsasl2_la_DEPENDENCIES = $(LTLIBOBJS) $(top_srcdir)/Versions
 
 libsasl2_la_LIBADD = $(SASL_DL_LIB) $(SASL_STATIC_LIBS) $(LIB_SOCKET) $(LIB_DOOR) $(PLUGIN_COMMON_OBJS)
 if BUILD_LIBOBJ
diff -up cyrus-sasl-2.1.28/Versions.1~ cyrus-sasl-2.1.28/Versions
--- cyrus-sasl-2.1.28/Versions.1~	2022-06-20 02:43:31.369214260 +0200
+++ cyrus-sasl-2.1.28/Versions	2022-06-20 02:43:31.369214260 +0200
@@ -0,0 +1,7 @@
+SASL2 {
+    global:
+        sasl_*; prop_*; auxprop_plugin_info; _sasl_MD5*;
+};
+
+HIDDEN { local: __*; _rest*; _save*; *; };
+
