diff -up cyrus-sasl-2.1.28/lib/Makefile.am.12~ cyrus-sasl-2.1.28/lib/Makefile.am
--- cyrus-sasl-2.1.28/lib/Makefile.am.12~	2022-06-20 03:24:47.924915010 +0200
+++ cyrus-sasl-2.1.28/lib/Makefile.am	2022-06-20 03:24:47.934915073 +0200
@@ -43,7 +43,7 @@
 # Library version info - here at the top, for sanity
 # See <http://www.gnu.org/software/libtool/manual/libtool.html#Versioning>
 # CURRENT:REVISION:AGE
-sasl_version = 3:0:0
+sasl_version = 2:25:0
 
 AM_CPPFLAGS=-DLIBSASL_EXPORTS=1 -I$(top_srcdir)/include -I$(top_srcdir)/plugins \
                -I$(top_builddir)/include -I$(top_srcdir)/sasldb -I$(top_srcdir)/common -DCONFIGDIR='"${configdir}"' -DPLUGINDIR='"${plugindir}"'
diff -up cyrus-sasl-2.1.28/plugins/Makefile.am.12~ cyrus-sasl-2.1.28/plugins/Makefile.am
--- cyrus-sasl-2.1.28/plugins/Makefile.am.12~	2022-06-20 03:24:47.934915073 +0200
+++ cyrus-sasl-2.1.28/plugins/Makefile.am	2022-06-20 03:28:54.216470850 +0200
@@ -47,7 +47,7 @@
 # Library version info - here at the top, for sanity
 # See <http://www.gnu.org/software/libtool/manual/libtool.html#Versioning>
 # CURRENT:REVISION:AGE
-plugin_version = 3:0:0
+plugin_version = 2:25:0
 
 AM_CPPFLAGS=-DLIBSASL_EXPORTS=1 -I$(top_srcdir)/include -I$(top_srcdir)/lib -I$(top_srcdir)/sasldb -I$(top_srcdir)/common -I$(top_builddir)/include
 AM_LDFLAGS = -module -export-dynamic -rpath $(plugindir) -version-info $(plugin_version) -no-undefined
