diff -up cyrus-sasl-2.1.28/configure.ac.13~ cyrus-sasl-2.1.28/configure.ac
--- cyrus-sasl-2.1.28/configure.ac.13~	2022-06-20 03:29:18.486624162 +0200
+++ cyrus-sasl-2.1.28/configure.ac	2022-06-20 03:29:59.770218277 +0200
@@ -1497,7 +1497,6 @@ AH_BOTTOM([#if defined __GNUC__ &&  __GN
 AC_CONFIG_HEADERS(config.h)
 
 AC_CONFIG_FILES([Makefile
-libsasl2.pc
 include/Makefile
 saslauthd/Makefile
 sasldb/Makefile
diff -up cyrus-sasl-2.1.28/Makefile.am.13~ cyrus-sasl-2.1.28/Makefile.am
--- cyrus-sasl-2.1.28/Makefile.am.13~	2022-02-18 22:53:25.000000000 +0100
+++ cyrus-sasl-2.1.28/Makefile.am	2022-06-20 03:29:18.483290807 +0200
@@ -44,6 +44,15 @@ ACLOCAL_AMFLAGS = -I m4
 #
 ################################################################
 
+do_subst = sed -e 's,[@]LIB_DOOR[@],$(LIB_DOOR),g' \
+	       -e 's,[@]SASL_DL_LIB[@],$(SASL_DL_LIB),g' \
+	       -e 's,[@]LIBS[@],$(LIBS),g' \
+	       -e 's,[@]VERSION[@],$(VERSION),g' \
+	       -e 's,[@]libdir[@],$(libdir),g' \
+	       -e 's,[@]prefix[@],$(prefix),g' \
+	       -e 's,[@]exec_prefix[@],$(exec_prefix),g' \
+	       -e 's,[@]includedir[@],$(includedir),g'
+
 if SASLAUTHD
 SAD = saslauthd
 else
@@ -136,6 +145,10 @@ pkgconfig_DATA = libsasl2.pc
 ## define this unconditionally because dist-hook references it
 SPHINX_CACHE = docsrc/.doctrees
 
+libsasl2.pc:	$(srcdir)/libsasl2.pc.in Makefile
+	rm -f $@
+	$(do_subst) < $(srcdir)/libsasl2.pc.in > $@
+
 dist-hook:
 	@find $(distdir) -exec chmod u+w {} ';'
 	@find $(distdir) -name CVS -print | xargs -t rm -rf
