Index: cyrus-sasl2-2.1.25.dfsg1/lib/common.c
===================================================================
--- cyrus-sasl2-2.1.25.dfsg1.orig/lib/common.c	2014-02-10 17:45:24.000000000 -0600
+++ cyrus-sasl2-2.1.25.dfsg1/lib/common.c	2014-02-10 17:47:36.379878155 -0600
@@ -841,7 +841,7 @@
 
   if (!free_mutex || free_mutex == 0x1)
     free_mutex = sasl_MUTEX_ALLOC();
-  if (!free_mutex) return SASL_FAIL;
+  if (!free_mutex) return;
 
   result = sasl_MUTEX_LOCK(free_mutex);
   if (result!=SASL_OK) return;
