--- saslauthd/auth_rimap.c	2009-12-03 14:07:03.000000000 -0500
+++ saslauthd/auth_rimap.c.oden	2011-10-14 12:26:39.109464650 -0400
@@ -164,6 +164,7 @@ qstring (
     while ((p1 = strchr(p1, '"')) != NULL) {
 	p1++;
 	num_quotes++;
+	++p1;
     }
     
     if (!num_quotes) {
@@ -199,7 +200,7 @@ qstring (
 	}
 	*p2++ = *p1++;
     }
-    strcat(p2, "\"");
+    strcpy(p2, "\"");
     return c;
 }
 
