--- plugins/Makefile.am	2011-10-14 14:19:49.319460683 -0400
+++ plugins/Makefile.am.oden	2011-10-14 14:27:16.149460630 -0400
@@ -49,7 +49,7 @@
 plugin_version = 2:25:0
 
 INCLUDES=-I$(top_srcdir)/include -I$(top_srcdir)/lib -I$(top_srcdir)/sasldb -I$(top_builddir)/include
-AM_LDFLAGS = -module -export-dynamic -rpath $(plugindir) -version-info $(plugin_version)
+AM_LDFLAGS = -module -export-dynamic -rpath $(plugindir) -avoid-version
 
 COMPAT_OBJS = @LTGETADDRINFOOBJS@ @LTGETNAMEINFOOBJS@ @LTSNPRINTFOBJS@
 
